gamemodes/terrortown/gamemode/client/cl_awards.lua

```diff
@@ -177,48 +181,60 @@ end
 FirstBlood = AWARDS.FirstBlood -- just for compatibility
 
 ---
+-- See if there is one killer responsible for all kills of either team
 -- @param table events
 -- @param table scores
 -- @param table players list of @{Player}s with key = steamid64 and value = nickname of the @{Player}
 -- @param table traitors list of @{Player}s with key = steamid64 and value = nickname of the @{Player}
 -- @realm client
 function AWARDS.AllKills(events, scores, players, traitors)
-	-- see if there is one killer responsible for all kills of either team
-
-	local killed_traitors = {}
-	local killed_not_traitors = {}
+	local killedTraitors, killedNoTraitors
```

Comment:
Since the naming convention for state functions is `IsState` renaming this to `hasKilledTraitors` might be a good idea. But I don't know, please discuss.

Edit: Okay, after reading through the code it seems to be no boolean. But I'm still confused with the name of this variable. It sounds like a table of players? But it doesn't seem to be a table

Reply:
What do you wanna say with this comment? :D

Reply:
> It sounds like a table of players? But it doesn't seem to be a table

The naming of this variable does not explain to me what it does

Reply:
`killedTraitors` stores the id of the player that killed all traitors (as not traitor), `killedNoTraitors` stores the id of the player that killed no traitors (as traitor).

Reply:
hm, I still don't like the variable names. But I have no better idea. So it is ok for me. Maybe add this explanation as a comment?